nemo/collections/nlp/__init__.py

```diff
@@ -15,8 +15,6 @@
 
 import nemo
 from .data import *
```

Comment:
If you are going to do a nlp refactor, can you remove as much of these relative imports as you can?
Eg change this line from `from .data import *` to `from nemo.collections.nlp.data import *`, get rid of the "."

Reply:
ok will do, thanks for the quick feedback! Let me know if you see some other things